.github/workflows/persubmit.yml

```
@@ -19,12 +19,12 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
-      - name: Download Pre-Built LLVM 8.0.1
+      - name: Download Pre-Built LLVM 10.0.0
         run: |
           mkdir taichi-llvm
           cd taichi-llvm
-          wget https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1-linux-x64.zip --waitretry=3 --tries=5
-          unzip taichi-llvm-8.0.1-linux-x64.zip
+          wget https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0-linux.zip --waitretry=3 --tries=5
+          unzip taichi-llvm-10.0.0-linux.zip
```

Cool! IIUC now all CI are running on LLVM 10? So will we deprecate LLVM 8 very soon? If not, you may try this:
```suggestion
          unzip taichi-llvm-${{ matrix.llvm }}-linux.zip
```
and update the `strategy` of course, WDYT?

---

Yeah, we will deprecate LLVM 8 very soon, so I'd just keep things simple here :-) Thanks for the review.